packages/embarkjs-ipfs/src/index.js

```diff
@@ -1,4 +1,4 @@
-const IpfsApi = require('ipfs-api');
+const IPFS = require('ipfs-http-client');
```

Comment:
One small thing, not really a PR blocker. Usually in `.js` and `.ts` all caps is used for constants, so it might be a little confusing to have a "library variable" that looks like a constant later on in the code.

Reply:
I'm using all constants because `IPFS` is an acronym, like `HTTP` and `TCP`. This is also, technically, a constant 😉

Reply:
I'd go with Michael here. I'd even rename it `ipfsClient` so it's clear what it is.